workspace/api/openfdd_bridge/feather_store.py

```
@@ -108,15 +108,33 @@ def _column_list(columns: list[str] | str | None) -> list[str] | None:
 def _apply_arrow_thread_env() -> None:
-    n = _env_int("OFDD_ARROW_IO_THREADS", 0)
-    if n <= 0 or not _USE_FEATHER:
+    if not _USE_FEATHER:
         return
     try:
-        import pyarrow as pa
+        from open_fdd.arrow_runtime.config import configure_arrow_runtime
 
-        pa.set_cpu_count(n)
+        configure_arrow_runtime()
     except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)